terraform/projects/infra-jump-instance/README.md

```diff
@@ -0,0 +1,56 @@
+## Project: infra-jump-instance
+
+This project creates a jump box which enables you access into your AWS
+private subnets.
+
+When auto scaling groups are deployed into the private they are not accessible
+from public networks since they do not have a public IP address. In order to
+access them a jump box is in created from within this project. The instance has
+got a public IP address which can be used in order to access.
+
+There is a number of firewall rules that have been created to allow you access to all
+public + private subnets on any port. The instance is currently only allowed ssh access
+to port number 22 from any public network.
+
+In order to properly utilize this instance, you need to enable ssh-agent key forwarding.
```

Comment:
Wondering if all instructions to get this working should be in the README as it feels like someone might not know to look here?

Reply:
Don't think these instructions are used, there are instructions in the main `README` that explains how to use the jumpbox